meta/main.yml

```diff
@@ -20,6 +20,12 @@ galaxy_info:
       versions:
         - 7
 
+    - name: Ubuntu
+      versions:
+        - 16.04
+        - 18.04
+        - 20.04
```

Comment:
The Ubuntu release names are different. See [here](https://galaxy.ansible.com/api/v1/platforms/?name__icontains=ubuntu), they use e.g. **bionic**.

Reply:
amended commit and resolved the issue